dart/dynamics/InverseKinematics.h

```diff
@@ -478,7 +559,7 @@ class InverseKinematics : public common::Subject
 
     /// Constructor
     explicit JacobianTranspose(InverseKinematics* _ik,
-                               double _clamp = DefaultIKGradientComponentClamp);
+                               const Properties& properties = Properties());
```

Comment:
This will break API compatibility. Is this pull request for the next major version up? If so, please ignore this comment.